privateSBOMExchange/config/config.ini

```diff
@@ -0,0 +1,2 @@
+[database]
+url = sqlite:///PetraSBOMs.db
```

Comment:
can we, for the sake of efficiency and testing, use a [`mem`  connector](https://www.sqlite.org/inmemorydb.html). It will also keep things neater

Reply:
like this?